IHP/View/CSSFramework.hs

```diff
@@ -93,52 +93,104 @@ instance Default CSSFramework where
-                                            ! A.class_ (cs $ classes ["form-check-input", (inputInvalidClass, isJust validatorResult), (fieldClass, not (null fieldClass))])
-                                            !? (required, A.required "required")
-                                            !? (autofocus, A.autofocus "autofocus")
-                                            !? (fieldValue == "yes", A.checked "checked")
-                                            !? (disabled, A.disabled "disabled")
-                                    theInput
-                                    H.input ! A.type_ "hidden" ! A.name fieldName ! A.value (cs $ inputValue False)
-                                    Blaze.text fieldLabel
-                                    validationResult
-                                    helpText
-
-                    renderTextField :: Blaze.AttributeValue -> FormField -> Blaze.Html
+                    renderCheckboxFormField formField@(FormField {fieldType, fieldName, fieldLabel, fieldValue, fieldInputId, validatorResult, fieldClass, disabled, disableLabel, disableValidationResult, fieldInput, labelClass, required, autofocus }) = do
+                        formGroup [hsx|<div class="form-check">{element}</div>|]
+                        where
+                            theInput = [hsx|
+                                            <input
+                                                type="checkbox"
+                                                name={fieldName}
+                                                class={classes ["form-check-input", (inputInvalidClass, isJust validatorResult), (fieldClass, not (null fieldClass))]}
+                                                id={fieldInputId}
+                                                checked={fieldValue == "yes"}
+                                                required={required}
+                                                disabled={disabled}
+                                                autofocus={autofocus}
+                                            />
+
+                                            <input type="hidden" name={fieldName} value={inputValue False} />
+                                    |]
+
+
+                            element = if disableLabel
+                                then [hsx|<div>
+                                            {theInput}
+                                            {validationResult}
+                                            {helpText}
+                                        </div>
+                                    |]
+                                else [hsx|
+                                        {theInput}
+                                        <label
+                                            class={classes [("form-check-label", labelClass == ""), (labelClass, labelClass /= "")]}
+                                            for={fieldInputId}
```

Comment:
On `master` the `for` is missing. Was this for a reason?

Reply:
Likely a bug :)